README.md

```diff
@@ -346,6 +346,9 @@ All the configuration options can be changed globally via `configure` or per log
 * `:max_depth` - the maximum depth for nested structured data. Defaults to `3`.
 * `:metadata` - the meta info to display before the message, can be `:pid`, `:date`, `:time` or `:file`. Defaults to empty array `[]`, no metadata. Setting this to `:all` will print all the metadata.
 * `:types` - the new custom log types. Defaults to `{}`.
+* `:date_format` - uses `strftime` format to display dates. Defaults to `"%F"`.
+* `:time_format` - uses `strftime` format to display times. Defaults to `"%T.%3N"`.
```

Comment:
I went ahead and added date_format and time_format to the docs while I was in here. feel free to suggest we remove

Reply:
Great addition!